### DISCUSSION TITLE
Is anyone else concerned about your code being sent, and stored, by Microsoft and OpenAI?

### DISCUSSION THREAD

**Discussion by kmandeville** [community/community#13894] [Copilot Conversations]
We're already obviously keeping our code in Github and trusting them. But this plugin, which does sound cool, is now sending code snippets as well as whole other files, to Github Copilot team, Microsoft and OpenAI. Do you just innately trust all those other teams to safe guard your code? Hopefully you aren't using it in your companies without them signing off on this

**Comment by danmaxis** [comment]
Yes, I'm concerned about how have real productivity gains without exposing sensitive source code.
